src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -674,9 +688,10 @@ internal static bool IsFieldOrPropertyInitializer(BoundStatement initializer)
         {
             var syntax = initializer.Syntax;
 
-            if (syntax is ExpressionSyntax && syntax?.Parent.Kind() == SyntaxKind.EqualsValueClause) // Should be the initial value.
+            if (syntax is ExpressionSyntax && syntax?.Parent!.Kind() == SyntaxKind.EqualsValueClause) // Should be the initial value.
```

Comment:
Did't we add a `RequiredParent` helper, at least for the IDE? We should probably use it here as well.

Reply:
There is no such thing in the Compiler solution.

---
In reply to: [389181056](https://github.com/dotnet/roslyn/pull/42068#discussion_r389181056) [](ancestors = 389181056)